tests/ai_safety/nemo_guardrails/conftest.py

```
@@ -467,61 +538,280 @@ def verify_guardrails_healthcheck(
 
 @pytest.fixture(scope="class")
 def nemo_guardrails_llm_judge_healthcheck(
+    admin_client: DynamicClient,
     nemo_guardrails_llm_judge: NemoGuardrails,
     nemo_guardrails_llm_judge_route: Route,
     current_client_token: str,
-    openshift_ca_bundle_file: str,
 ) -> None:
     verify_guardrails_healthcheck(
         route=nemo_guardrails_llm_judge_route,
         openshift_ca_bundle_file=openshift_ca_bundle_file,
```

Flake8 (7.3.0)

[error] undefined name 'openshift_ca_bundle_file'

(F821)